src/main/java/com/puppycrawl/tools/checkstyle/checks/UncommentedMainCheck.java

```diff
@@ -52,6 +52,20 @@
  * <pre>
  * &lt;module name=&quot;UncommentedMain&quot;/&gt;
  * </pre>
+ * <p>Example:</p>
+ * <pre>
+ * public class Game {
+ *    public static void main(String[] args){}   // violation
```

Comment:
please add example of method that is named "main" but is not public or not static or parameters are not array of string.

Please add example of modern style "String... args":
https://stackoverflow.com/a/8756966/1015848